app/assets/styles/components/subscriptions.css

```diff
@@ -0,0 +1,22 @@
+.subscription__status {
```

Comment:
Having a `__status` element, means that we must have a `subscription` block somewhere that this element is a descendant of. Since this status lives on its own, it can just be `.subscription-status`